AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStoreAdmin/Model/KeyStoreAdmin.smithy

```diff
@@ -353,94 +386,42 @@ structure ApplyMutationOutput {
   MutatedBranchKeyItems: MutatedBranchKeyItems
 }
 
-// @documentation(
-//   "If the inputs align with a Mutation that is marked as already in-flight,
-//   this operation returns a Mutation Token that can be used to continue the in-flight
-//   Mutation.
-//   If no in-flight Mutation is detected, nothing is returned.
-//   If the inputs do not align, a MutationConflictException is thrown.")
-// operation ResumeMutation {
-//   input:  ResumeMutationInput
-//   output: ResumeMutationOutput
-//   errors: [MutationLockDisagreesException, KeyStoreAdminException]
-// }
-
-// @error("client")
-// @documentation("Mutation Lock does not agree with the provided Branch Key Properities.")
-// structure MutationLockDisagreesException {
-//   @required
-//   message: String,
-// }
-// // TODO: verify version before release
-// @documentation("
-// Define the Mutatable Properities of a Branch Key.
-// As of v1.8.0, the Mutable Properities are:
-// - The KmsArn protecting the Branch Key
-// - The custom encryption context of a Branch Key")
-// structure MutableBranchKeyProperities {
-//   @required
-//   @documentation("The KmsArn protecting the Branch Key.")
-//   KmsArn: String // KMS Arn validation MUST occur in Dafny
-//   @required  
-//   @documentation("The custom Encryption Context authenicated with this Branch Key.")
-//   CustomEncryptionContext: aws.cryptography.keyStore#EncryptionContextString // EC non Empty MUST be validated in Dafny
-// }
-
-
-// structure ResumeMutationInput {
-//   @documentation("The identifier for the Branch Key.")
-//   @required
-//   Identifier: String
-
-//   @documentation(
-//   "Describes the original mutable branch key properities.
-//   All members of the input MUST be supplied,
-//   or the operation will return an exception.")
-//   @required
-//   Original: MutableBranchKeyProperities
+// TODO: verify version before release
+@documentation("
+Define the Mutatable Properities of a Branch Key.
+As of v1.8.0, the Mutable Properities are:
+- The KmsArn protecting the Branch Key
+- The custom encryption context of a Branch Key")
+structure MutableBranchKeyProperities {
+  @required
+  @documentation("The KmsArn protecting the Branch Key.")
+  KmsArn: String // KMS Arn validation MUST occur in Dafny
```

Comment:
why not use the type that already exists in the smithy models?
`KmsKeyId`

Reply:
Ahh... for that, see the [KMS Developer Guide's Key identifiers](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id).

Really, this should be `KmsKeyArn`. 

Or it could be the same as the `KmsSymmetricEncryptionKeyARN`.